src/main/java/reactor/core/publisher/Flux.java

```diff
@@ -87,6 +87,8 @@
  */
 public abstract class Flux<T> implements Publisher<T> {
 
+	public interface Operator<I, O> extends Function<Subscriber<? super I>, Subscriber<? super O>> {}
```

Comment:
I'd rather name the arguments explicitly:

```java
Operator<Upstream, Downstream> extends Function<Subscriber<? super Downstream>, Subscriber<? super Upstream>>
```

The function takes the downstream's Subscriber and transforms it into something that will be subscribed to the upstream.